### ISSUE TITLE
Front end setting to make page full-width

### ISSUE THREAD

**Issue by mortenpi** [JuliaDocs/Documenter.jl#1168]
Sometimes the default max width of the main body column is too narrow, so it would be good if there was a way to make it wider, but without compromising readability in most cases.

* Have an option in the new settings dialog to disable the `max-width` on the main `<article>` tag. Implementation: apply a class e.g. `<html>` when that is toggled, similar to how we handle themes. Store the option in Web Storage.

* Also maybe have a keyboard shortcut for this, so that it would be easy to toggle (for people who know/want this). However, it's not really something you could rapidly toggle on-off I think since the reflow of the page will throw you to a different section of the page if you are somewhere in the middle.

Based on a suggestion by @KristofferC on Slack, writing it up so it would not get completely forgotten.

**Comment by brianguenter** [comment]
This has been idle for a while. Is anything happening? I'd also like this capability.

**Comment by mortenpi** [comment]
Nope, it's not really on the priority list, but happy to review a PR if someone wants to take a stab here. I'm okay with having some sort of an UI toggle for this.
